packages/graphql-server/src/createGraphQLYoga.ts

```diff
@@ -99,7 +99,7 @@ export const createGraphQLYoga = ({
   }`
 
     // TODO: Once Studio is not experimental, can remove these generateGraphiQLHeaders
-    const authHeader = `{"x-auth-comment": "See documentation: https://redwoodjs.com/docs/cli-commands#setup-graphiQL-headers on how to auto generate auth headers"}`
+    const authHeader = `{"x-auth-comment": "See documentation: https://redwoodjs.com/docs/cli-commands#setup-graphiqL-headers on how to auto generate auth headers"}`
```

Comment:
I don't think your link is correct. Is it not:

https://redwoodjs.com/docs/cli-commands#setup-graphiql-headers

You have a capital "L".

Reply:
yes I fix that